### PR TITLE
docs: User data handling

### PR DESCRIPTION
**Pull request by tumido**
Partials for: https://github.com/operate-first/support/issues/48 https://github.com/operate-first/apps/issues/100

This PR is a template, without the actual implementation details or a guide. I think that should follow up in separate PRs. I think we just need to have the issue templates in place, even without the actual how-to, so users can start filing in tickets.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pvc_request.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+about: "Template for users of JupyterHub requesting HW quota upgrade: disk size, CPU/GPU/Memory."
+title: ""
+labels: user-support
+assignees: ""
+---
+
+### Questionnaire
+
+1. **User details**:
+   Your MOC user name and which applications deployed by our team you're planning to use the bucket at
```
Comment (anishasthana) [inline .github/ISSUE_TEMPLATE/pvc_request.md:12]: This doesn't match the template

Comment (anishasthana) [reply .github/ISSUE_TEMPLATE/pvc_request.md:12]: Isn't the application here "jupyterhub"?

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/pvc_request.md (hunk 1 above)
Comment (tumido) [reply .github/ISSUE_TEMPLATE/pvc_request.md:12]: That's because I'm bad at copy pasting.. :smile:

**Review by anishasthana — COMMENTED** [review]
What do you think of changing HW to hardware everywhere in the doc? Just a thought, definitely not a blocking thing

**Comment by tumido** [comment]
@anishasthana done! :+1: :slightly_smiling_face:

File: docs/user_profiles.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Managing user profiles
```
Comment (4n4nd) [inline docs/user_profiles.md:1]: can we rename this file to `odh_user_profiles.md`?

Comment (4n4nd) [reply docs/user_profiles.md:1]: or nvm this includes opf stuff like ceph buckets so a general `user_profiles.md` works too

**Review by anishasthana — APPROVED** [review]
The fact that it's still complaining despite the lgtm seems silly :-)

**Review by HumairAK — APPROVED** [review]

**Pull request merged**